03-Make/soup/Makefile

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/make
```

Comment:
Shabang with /usr/bin/make makes command line:
/usr/bin/make ./Makefile
It just use ./Makefile as target. 
If you want use this Makefile as source of rules first line should be
#!/usr/bin/make -f